src/fundamental/logarithm.h

```diff
@@ -30,6 +28,15 @@ static inline unsigned u32ctz(uint32_t n) {
 #endif
 }
 
+#define popcount(n)                                             \
+        _Generic((n),                                           \
+                 short unsigned: __builtin_popcount(n),         \
```

Comment:
hmm, I think most people what call the type "unsigned short" rather than "short unsigned"...

Reply:
I mean, you even use "unsigned long" in this order just two lines below...

Reply:
what about "unsigned char" btw? just for completeness sake?